bugbug/models/stepstoreproduce.py

```diff
@@ -92,6 +111,15 @@ def get_labels(self):
 
         return classes, [0, 1]
 
+    def rollback_gen(self, bug, classes):
+        for change in classes["to_rollback"][int(bug["id"])]:
+            self.rollback_change = change
+            rollbacked_bug = bug_snapshot.rollback(bug, self.rollback_when)
+            if change["field_name"] == "cf_has_str" and change["removed"] == "no":
+                return rollbacked_bug, 0
```

Comment:
@marco-c now we can rollback the same bug multiple times , there's this one thing to fix though , we need to find a way to "yield" instead of "return".

Reply:
Also , i can move rollback_gen function to model.py if that's better.